java/core/src/main/java/sleeper/core/partition/PartitionTree.java

```diff
@@ -150,6 +165,20 @@ public List<Partition> getLeafPartitions() {
         return idToPartition.values().stream().filter(Partition::isLeafPartition).collect(toUnmodifiableList());
     }
 
+    /**
+     * Expands the leaf partition call to only return the ids of the partitions.
+     *
+     * @return list containing all the leaf partition ids
+     */
+    public List<String> getLeafPartitionIds() {
```

Comment:
It looks like this is only used in tests. Please either move it to the test code, or add a step to the assertions to extract the partition ID.